authors/apps/core/exceptions.py

```diff
@@ -52,3 +53,10 @@ def _handle_rating_not_found(exc, context, response):
     response = Response({'errors': 'rating not found'}, status=status.HTTP_404_NOT_FOUND)
```

Comment:
I would recommend shifting the error message variables to another file so you can have a single reference for messages when testing as well.

Reply:
done